app/models/concerns/sufia/ability.rb

```diff
@@ -63,7 +63,9 @@ def feature_abilities
     end
 
     def admin_set_abilities
-      can :create, AdminSet if admin?
+      can :manage, AdminSet if admin?
```

Comment:
What would you think about batching these up? Like:

```ruby
can :manage, [AdminSet, Sufia::PermissionTemplate, Sufia::PermissionTemplateAccess] if admin?
```

Or put all three abilities in an `if admin?` block?